src/js/components/Avatar/stories/typescript/Stacked.tsx

```diff
@@ -67,7 +66,7 @@ export const Stacked = () => {
         {/* Notification */}
         <Stack anchor="top-right">
           <Avatar src={shimi} />
-          <Box pad="xsmall" round background="accent-4" responsive={false} />
+          <Box pad="xsmall" round background="black-4" responsive={false} />
```

Comment:
```suggestion
          <Box pad="xsmall" round background="light-1" responsive={false} />
```
I don't think black-4 is a color